fundamentals/android/auth-sign-in-with-google/Notes/app/src/main/java/com/notes/app/screens/authentication/sign_in/SignInScreen.kt

```diff
@@ -36,17 +38,21 @@ import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.hilt.navigation.compose.hiltViewModel
 import com.notes.app.R
-import com.notes.app.screens.account_center.AuthenticationButton
+import com.notes.app.screens.authentication.AuthenticationButton
+import com.notes.app.screens.authentication.launchCredManBottomSheet
 import com.notes.app.ui.theme.NotesTheme
 import com.notes.app.ui.theme.Purple40
 
 @Composable
 @OptIn(ExperimentalMaterial3Api::class)
 fun SignInScreen(
+    openScreen: (String) -> Unit,
```

Comment:
At first glance, it's not clear what `openScreen()` does? Should we use a more descriptive name?
Given that this is a callback, we might also want to use the `on` prefix, so maybe name it `onSignUpClicked` ?

Reply:
Changed to `onSignUpClicked`